crates/nu-parser/src/parse_keywords.rs

```diff
@@ -1831,32 +1831,39 @@ pub fn parse_module_block(
     (block, module, module_comments)
 }
 
-fn parse_module_file(
-    working_set: &mut StateWorkingSet,
-    path: ParserPath,
-    path_span: Span,
-    name_override: Option<String>,
-) -> Option<ModuleId> {
-    if let Some(i) = working_set
-        .parsed_module_files
-        .iter()
-        .rposition(|p| p == path.path())
-    {
-        let mut files: Vec<String> = working_set
-            .parsed_module_files
+/// If importing the script `path` would cause a circular import, returns true and adds a ParseError to the WorkingSet.
+fn is_circular_import(ws: &mut StateWorkingSet, path: &ParserPath, path_span: &Span) -> bool {
```

Comment:
It's better to use `working_set` to keep the naming consistent.

Reply:
Sure.